owlbot.py

```diff
@@ -0,0 +1,192 @@
+    s.replace(
+      "dev/src/v1/firestore_client.ts",
+      "/protos/protos'",
+      "/protos/firestore_v1_proto_api'"
+    )
+    s.replace(
+      "dev/test/gapic_firestore_v1.ts",
+      "/protos/protos'",
+      "/protos/firestore_v1_proto_api'"
+    )
+    s.replace(
+      "dev/test/gapic_firestore_v1.ts",
+      "import \* as firestoreModule from '\.\./src';",
+      "import * as firestoreModule from '../src/v1';"
+    )
+    s.replace(
+      "dev/test/gapic_firestore_v1.ts",
+      "firestoreModule\.v1",
+      "firestoreModule"
+    )
+    s.replace(
+      "dev/src/v1/firestore_admin_client.ts",
+      "/protos/protos'",
+      "/protos/firestore_admin_v1_proto_api'"
+    )
+    s.replace(
+      "dev/test/gapic_firestore_admin_v1.ts",
+      "/protos/protos'",
+      "/protos/firestore_admin_v1_proto_api'"
+    )
+    s.replace(
+      "dev/test/gapic_firestore_admin_v1.ts",
+      "import \* as firestoreadminModule from '\.\./src';",
+      "import * as firestoreadminModule from '../src/v1';"
+    )
+    s.replace(
+      "dev/test/gapic_firestore_admin_v1.ts",
+      "firestoreadminModule\.v1",
+      "firestoreadminModule"
+    )
+    s.replace(
+      "dev/src/v1beta1/firestore_client.ts",
+      "/protos/protos'",
+      "/protos/firestore_v1beta1_proto_api'"
+    )
+    s.replace(
+      "dev/test/gapic_firestore_v1beta1.ts",
+      "/protos/protos'",
+      "/protos/firestore_v1beta1_proto_api'"
+    )
+    s.replace(
+      "dev/test/gapic_firestore_v1beta1.ts",
+      "import \* as firestoreModule from \'../src\';",
+      "import * as firestoreModule from '../src/v1beta1';"
+    )
+    s.replace(
+      "dev/test/gapic_firestore_v1beta1.ts",
+      "firestoreModule\.v1beta1",
+      "firestoreModule"
+    )
+
+    # Mark v1beta1 as deprecated
+    s.replace(
+      "dev/src/v1beta1/firestore_client.ts",
+      "@class",
+      "@class\n * @deprecated Use v1/firestore_client instead."
+    )
+    s.replace(
+      "dev/src/v1beta1/firestore_client.ts",
+      "const version",
+      "// tslint:disable deprecation\n\nconst version",
+      1
+    )
+
+    os.rename("dev/.gitignore", ".gitignore")
+    os.rename("dev/.eslintignore", ".eslintignore")
+    os.rename("dev/.mocharc.js", ".mocharc.js")
+    os.rename("dev/.jsdoc.js", ".jsdoc.js")
+    os.rename("dev/.prettierrc.js", ".prettierrc.js")
+    os.unlink("dev/.eslintrc.json")
+
+    s.replace(".jsdoc.js", "protos", "build/protos", 1)
+
+    # Remove auto-generated packaging tests
+    os.system('rm -rf dev/system-test/fixtures dev/system-test/install.ts')
+
+    os.chdir("dev")
+    node.compile_protos_hermetic()
+    os.chdir("protos")
+    os.unlink('protos.js')
+    os.unlink('protos.d.ts')
+    subprocess.run('./update.sh', shell=True)
+    os.chdir("../../")
+
+    # Copy types into types/
+    # These files were generated by node.compile_protos_hermetic() above.
+    os.system("cp build/src/v1/firestore*.d.ts types/v1")
```

Comment:
@schmidt-sebastian I uncommented these lines that copied the `*.d.ts` files.  They're generated on line 153 above.  I think this pull request is good to merge.